.gitignore

```diff
@@ -0,0 +1,95 @@
+
```

Comment:
Can you add this `.gitignore` in a separate pull request? Or I could just add one after we finish this pull request.